### PR TITLE
Update ASP.NET workflow server and designer doc

### PR DESCRIPTION
**Pull request by agriffard**
Related to [#3664](https://github.com/elsa-workflows/elsa-core/discussions/3664)

https://v3.elsaworkflows.io/docs/installation/aspnet-apps-workflow-server-and-designer

I tried to follow the guide step by step and update the changes to target the V3 prerelease packages.

When I launch the app, I get a 404, though.

### CODE REVIEW

**Comment by agriffard** [comment]
Preview updated pages:
https://elsa-3-docs-git-fork-agriffard-main-elsa-workflows.vercel.app/docs/installation/packages
https://elsa-3-docs-git-fork-agriffard-main-elsa-workflows.vercel.app/docs/installation/aspnet-apps-workflow-server-and-designer

**Comment by agriffard** [comment]
Changes applied for Identity are the same than the ones suggested in #4.

File: v3/src/pages/docs/installation/aspnet-apps-workflow-server-and-designer.md [hunk 1]
```diff
@@ -22,24 +22,24 @@ dotnet new web -n "WorkflowsApp" -f net7.0
 CD into the project's root directory and add the following packages:
 
 ```shell
-cd WorkflowServer.Api
-dotnet add package Elsa
-dotnet add package Elsa.Http
-dotnet add package Elsa.Workflows.Api
-dotnet add package Elsa.Workflows.Designer
+cd WorkflowsApp
```
Comment (agriffard) [inline v3/src/pages/docs/installation/aspnet-apps-workflow-server-and-designer.md:25]: or keep `WorkflowServer.Api` but the folder name has to be the same than the app created previously.

**Comment by agriffard** [comment]
This commit https://github.com/elsa-workflows/elsa-documentation/commit/5b3af4079bb11d63135ee11a01c22a4be96ad787 omitted the second link that also had to be changed in the NuGet.config example

**Review by sfmskywalker — APPROVED** [review]
@agriffard It is good to see you here my friend. Thank you for the fixes 🙏🏻

**Pull request merged**